src/lib/node/outgoing_message.ts

```diff
@@ -1,5 +1,11 @@
+import { TranslatedValueID, ValueMetadata } from "zwave-js";
 import { NodeCommand } from "./command";
 
 export interface NodeResultTypes {
   [NodeCommand.setValue]: { success: boolean };
+  [NodeCommand.refreshInfo]: {};
+  [NodeCommand.getDefinedValueIDs]: { valueIds: TranslatedValueID[] };
```

Comment:
Not sure if we want to return object or directly `TranslatedValueID[]`.

Reply:
Always an object.